src/components/v2/appDetails/k8Resource/nodeDetail/NodeDetailTabs/Manifest.component.tsx

```diff
@@ -210,6 +256,7 @@ const ManifestComponent = ({
                         ),
                     _showDesiredAndCompareManifest &&
                         getDesiredManifestResource(appDetails, params.podName, params.nodeType, abortController.signal),
+                    handleInitializeGUISchema(abortController.signal),
```

Comment:
It is returing Promise<void> felt like an overkill to use Promise.allSettled here since on error of handleInitializeGUISchema we are already silently handling this